ui/dashboard/src/hooks/useDashboardWebSocket.ts

```diff
@@ -1,5 +1,5 @@
-import { ElementType, IActions } from "./useDashboard";
 import { useCallback, useEffect, useRef } from "react";
+import { ElementType, IActions } from "./useDashboard";
```

Comment:
I typically try to keep destructured imports alphabetical - given React is already in scope globally it's not critical to have them first.